src/components/ListBox.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="list-view">
+    <div class="list-icon-size">
+      <i :class="iconStyle" style="position: relative; color: rgb(70, 161, 189); alignment: center"></i>
```

Comment:
please move nested css attribute to `<script>` part.

Reply:
Moved to css attribute to `style` part